public/constant/content.ts

```diff
@@ -1,29 +1,33 @@
 const common = {
   ethTaipei: "ETHTaipei 2024 | March 21-24",
   ethTaipeiIntro:
-    "Welcome to the ETHTaipei 2024 event held in Taiwan. With a thriving Ethereum and developer community in Taiwan, ETHTaipei 2024 has brought together teams from around the world to participate in a 3-day hackathon and 2-day conference focusing on the application and technology of Ethereum.",
+    // "Welcome to the ETHTaipei 2024 event held in Taiwan. With a thriving Ethereum and developer community in Taiwan, ETHTaipei 2024 has brought together teams from around the world to participate in a 3-day hackathon and 2-day conference focusing on the application and technology of Ethereum.",
+    "Welcome to the ETHTaipei 2024 event held in Taiwan. With a thriving Ethereum and developer community in Taiwan, ETHTaipei 2024 has brought together teams from around the world to participate in a 2-day conference focusing on the application and technology of Ethereum.",
+  ethTaipeiIntro2:
+    // `At ETHTaipei, you'll have the chance to hear renowned experts, scholars, and industry leaders from around the world discuss the latest trends and applications of Ethereum. You can also participate in hackathons, workshops, and various other activities, fostering collaboration and idea exchange with developers worldwide to build valuable blockchain applications.`,
+    `At ETHTaipei, you'll have the chance to hear renowned experts, scholars, and industry leaders from around the world discuss the latest trends and applications of Ethereum. You can also participate in workshops and various other activities, fostering collaboration and idea exchange with developers worldwide to build valuable blockchain applications.`,
 };
 
 const homepage = {
-  hackathon: "Hackathon",
-  hackathonDate: "22-24 March, 2024",
+  // hackathon: "Hackathon",
+  // hackathonDate: "22-24 March, 2024",
   conference: "Conference",
   conferenceDate: "21-22 March, 2024",
   ethTaipeiIntroTitle: "What is ETHTaipei?",
   ethTaipeiIntro1: `Welcome to ETHTaipei the 2024 edition!!! With a vibrant Ethereum and developer community, ETHTaipei unites Ethereum-focused teams from across the globe, showcasing Taiwan as a significant hub for blockchain technology. This event also offers Taiwanese teams a platform to connect with the global community and gain increased visibility.`,
-  ethTaipeiIntro2: `At ETHTaipei, you'll have the chance to hear renowned experts, scholars, and industry leaders from around the world discuss the latest trends and applications of Ethereum. You can also participate in hackathons, workshops, and various other activities, fostering collaboration and idea exchange with developers worldwide to build valuable blockchain applications.`,
+  ethTaipeiIntro2: common.ethTaipeiIntro2,
   ethTaipeiIntro3_1: `ETHTaipei presents an unparalleled opportunity to learn about cutting-edge technology and applications about Ethereum (check out our 2023 edition `,
```

Comment:
Note: curious why intro 3 is separated into 2 part.

Reply:
ive no idea!